In [21]:
import numpy as np
import statistics as st

# Exercise 1

## Data
1. Greg 14
2. Marcia 12
3. Peter 11
4. Jan 10
5. Bobby 8
6. Oliver 8
7. Cindy 6


## Calculations
$ \sum x = 69$
<br> n = 7
<br> Mean $= \frac{69}{7} = 9.8571$
<br> Median = 10 (Jan)
<br> Mode = 8 (Bobby, Oliver)
<br> $ \sum  (x - \mu)^2 = \frac{314}{7} = 44.8571$
<br> Variance $= \frac{44.8571}{7} = 6.4081$
<br> Standard deviation $= \sqrt{6.4081} = 2.5314$
<br> Standard error $= \frac{2.5314}{\sqrt{7}} = .9567$ 


## Python Calculations

In [22]:
ages_1 =[14,12,11,10,8,8,6]

def stats(ages):
    print("mean: ",np.mean(ages))
    print("median: ",np.median(ages))
    print("variance: ",np.var(ages))
    print("std. deviation :",np.std(ages))
    print("standard error :",np.std(ages)/np.sqrt(len(ages)))

stats(ages_1)
print("Mode: 8")

mean:  9.857142857142858
median:  10.0
variance:  6.408163265306122
std. deviation : 2.531435020952764
standard error : 0.9567925036515135
Mode: 8


***
# Exercise 2

I would choose the mean of 9.857 (since it is fairly close to the median) and the standard deviation of 2.5314 

***
# Exercise 3

## Data
1. Greg 14
2. Marcia 12
3. Peter 11
4. Jan 10
5. Bobby 8
6. Oliver 8
7. Cindy 7


## Calculations
$ \sum x = 70$
<br> n = 7
<br> Mean $= \frac{70}{7} = 10$
<br> Median = 10 (Jan)
<br> Mode = 8 (Bobby, Oliver)
<br> $ \sum  (x - \mu)^2 = 38$
<br> Variance $= \frac{38}{7} = 5.4285$
<br> Standard deviation $= \sqrt{5.4285} = 2.3299$
<br> Standard error $= \frac{2.3299}{\sqrt{7}} = .8806$ 

## Changes
$ \sum x $
<br> Mean 
<br> $ \sum  (x - \mu)^2$
<br> Variance 
<br> Standard deviation 
<br> Standard error 

## No Change
n 
<br> Median 
<br> Mode 


## Python Calculations

In [23]:
ages_3 =[14,12,11,10,8,8,7]
stats(ages_3)
print("Mode is 8")

mean:  10.0
median:  10.0
variance:  5.428571428571429
std. deviation : 2.32992949004287
standard error : 0.8806305718527109
Mode is 8


***
# Exercise 4

## Data
1. Greg 14
2. Marcia 12
3. Peter 11
4. Jan 10
5. Bobby 8
6. Cindy 7
7. Jessica 1

## Calculations
$ \sum x = 63$
<br> n = 7
<br> Mean $= \frac{63}{7} = 9$
<br> Median = 10 (Jan)
<br> Mode = none
<br> $ \sum  (x - \mu)^2 = 108$
<br> Variance $= \frac{108}{7} = 15.4285$
<br> Standard deviation $= \sqrt{18} = 3.9279$
<br> Standard error $= \frac{3.9279}{\sqrt{7}} = 1.4846$ 

I would choose median of 10 (since there is outlier of 1) and standard deviation of 3.9279

## Python Calculations

In [24]:
ages_4 =[14,12,11,10,8,7,1]
stats(ages_4)
print("No Mode")

mean:  9.0
median:  10.0
variance:  15.428571428571429
std. deviation : 3.927922024247863
standard error : 1.4846149779161806
No Mode


***
# Exercise 5

## Data
1. TV Guide 20%
2. Entertainment Weekly 23%
3. Pop Culture Today 17% 
4. SciPhi Phanatic 5%

I would estimate 16.25% of adult americans were Brady Bunch fans based on all the sample magazines. I would also only use the first three smaples since SciPhi Phanatic seems like an outlier and a niche magazine, making the estamate to be 20%.


# Note

Standard Error is calculated as $ \frac{std. dev.}{\sqrt{n}}$ while in the answer sheet is calculated as $ \frac{std. dev.}{\sqrt{n-1}}$